test/jdk/java/time/test/java/time/TestLocalDate.java

```diff
@@ -420,6 +423,36 @@ public void test_toEpochDay_fromMJDays_symmetry() {
         }
     }
 
+    @Test
+    public void test_toEpochDay_edges() {
+        long minDay = ChronoField.EPOCH_DAY.range().getMinimum();
+        long maxDay = ChronoField.EPOCH_DAY.range().getMaximum();
+        long minYear = ChronoField.YEAR.range().getMinimum();
+        long maxYear = ChronoField.YEAR.range().getMinimum();
+        for (int i = 0; i < 500; i++) {
```

Comment:
The 500 seems like it would burn a lot of cycles mostly redundant.
How about { 1, 2, 3, 28, 29, 30, 31, 32, 363, 364, 365, 366, 367}.

Reply:
Yes, makes sense to avoid useless work. Added 0 to the list.